core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestServiceHandler.java

```diff
@@ -173,8 +173,8 @@ public S3RestServiceHandler(@Context ServletContext context)
               .setRecursive(true)
               .setMode(PMode.newBuilder()
                   .setOwnerBits(Bits.ALL)
-                  .setGroupBits(Bits.READ_EXECUTE)
-                  .setOtherBits(Bits.READ_EXECUTE).build())
+                  .setGroupBits(Bits.ALL)
```

Comment:
unit test case possible?

Reply:
Yes, but no. The test cases are really... fidgety for some reason. https://github.com/Alluxio/alluxio/pull/16180#issuecomment-1244163136

I'll be getting the unit tests in afterwards

Reply:
Let me try again

Reply:
fine to follow up on a separate PR if this is blocking patch progress/deadline. just wanted to be sure the idea crossed your mind

Reply:
and i totally missed the PR level comment -> jumped right into the code diff

Reply:
Changed the resolution here; managed to fix the unit test issue I had by using `MUST_CACHE`.